iib/web/models.py

```diff
@@ -1131,6 +1131,11 @@ def to_json(self, verbose=True):
             self.from_bundle_image_resolved, 'pull_specification', None
         )
         rv['organization'] = self.organization
+        if current_app.config['IIB_REQUEST_RELATED_BUNDLES_DIR']:
+            rv['related_bundles'] = {
+                'expiration': self.temporary_data_expiration.isoformat() + 'Z',
+                'url': url_for('.get_related_bundle_images', request_id=self.id, _external=True),
```

Comment:
I think this should be `.get_related_bundles` and the function in `api_v1.py` should be renamed to `get_related_bundles` as well. WDYT?

Reply:
Yes, indeed